app/views/projects/workflow_executions/_workflow_execution.html.erb

```diff
@@ -25,9 +25,7 @@
     },
     class: "text-slate-900 dark:text-slate-100 font-semibold hover:underline" %>
```

Comment:
**nitpick:** Can up update the first column as I said previously with `bg-slate-50 dark:bg-slate-900`

Reply:
Updated in 11361028a12f6069f80e16afbc750c6028be7147